std_msgs/UInt32 id
std_msgs/Int32 direction
std_msgs/Float64 x_position
std_msgs/Float64 y_position
std_msgs/Float64 speed
geometry_msgs/Point start_point
geometry_msgs/Point end_point
geometry_msgs/Point[] future_trajectory